CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changes
 
+* [#410](https://github.com/rubocop-hq/rubocop-rails/pull/410): Add new`Rails/TransactionRequiresNew` cop. ([@the-wendell][])
```

Comment:
This entry belongs to the `### New features` category. Can you create category and move to it?